static/app/components/performance/waterfall/treeConnector.tsx

```diff
@@ -21,21 +21,31 @@ export const ConnectorBar = styled('div')<{orphanBranch: boolean}>`
 type TogglerTypes = OmitHtmlDivProps<{
   hasToggler?: boolean;
   isLast?: boolean;
+  hasCollapsedSpanGroup?: boolean;
 }>;
 
 export const TreeConnector = styled('div')<TogglerTypes & {orphanBranch: boolean}>`
   height: ${p => (p.isLast ? ROW_HEIGHT / 2 : ROW_HEIGHT)}px;
   width: 100%;
-  border-left: 1px ${p => (p.orphanBranch ? 'dashed' : 'solid')} ${p => p.theme.border};
+  border-left: ${p => {
```

Comment:
I think there may be a slight visual bug when a leaf node of the span tree is part of a span grouping:
![image](https://user-images.githubusercontent.com/83961295/125864789-050dfe5b-c0c2-4acd-bf1c-84a82961613a.png)
the tree graphic doesn't connect to the node :(

Reply:
Should be fixed.